engine/api/src/main/java/io/deephaven/engine/table/ModifiedColumnSet.java

```diff
@@ -102,7 +102,7 @@ public boolean containsAny(ModifiedColumnSet columnSet) {
 
         @Override
         public boolean containsAll(ModifiedColumnSet columnSet) {
-            return columnSet.equals(this);
+            return true;
```

Comment:
What is `columnSet` is empty?

Reply:
Fair. I guess I was hung up on `containsAll` being true implies `containsAny` to also be true, but that is not so for this implementation. I do think the definition of `containsAny` seems correct as-is.